src/rendering/renderers/gpu/pipeline/PipelineSystem.ts

```diff
@@ -312,6 +313,13 @@ export class PipelineSystem implements System
             {
                 const attribute = geometry.attributes[i];
 
+                if ('divisor' in attribute)
+                {
+                    // TODO: Maybe emulate divisor with storage_buffers/float_textures?
+                    // For now just issue a warning
+                    warn(`Attribute ${i} has 'divisor' field set, which is currently unsupported in WebGPURenderer.`);
```

Comment:
this is great! i think we should trow this warning only if the divoser is not 1.

`Attribute ${i} has an invalid 'divisor' value of 'X'. WebGPU only supports a divisor value of 1.`